### PR TITLE
configure regulators as secure and invoke soc.c

### PR DESCRIPTION
**Pull request by SebastianBoe**
Change-Id: I9bc7f2b158c0ad9da0c434954c9619da5b70d754

### CODE REVIEW

**Review by vili-nordic — APPROVED** [review]

File: platform/ext/target/nordic_nrf/common/core/target_cfg.c [hunk 1]
```diff
@@ -1142,6 +1148,13 @@ enum tfm_plat_err_t spu_periph_init_cfg(void)
 		spu_peripheral_config_secure(base_addresses[i], SPU_LOCK_CONF_LOCKED);
 	}
 
+	/* Configure NRF_REGULATORS, and NRF_OSCILLATORS to be secure as NRF_REGULATORS.POFCON is needed
+	 * to prevent glitches when the power supply is attacked.
+	 *
+	 * NB: Note that NRF_OSCILLATORS and NRF_OSCILLATORS have the same base address and must therefore
```
Comment (mswarowsky) [inline platform/ext/target/nordic_nrf/common/core/target_cfg.c:1154]:
```suggestion
	 * NB: Note that NRF_OSCILLATORS and NRF_REGULATORS have the same base address and must therefore
```
?

**Review by mswarowsky — APPROVED** [review]
Small nit in comment, but else fine for me

File: platform/ext/target/nordic_nrf/common/core/target_cfg.c [hunk 2]
```diff
@@ -1142,6 +1148,13 @@ enum tfm_plat_err_t spu_periph_init_cfg(void)
 		spu_peripheral_config_secure(base_addresses[i], SPU_LOCK_CONF_LOCKED);
 	}
 
+	/* Configure NRF_REGULATORS, and NRF_OSCILLATORS to be secure as NRF_REGULATORS.POFCON is needed
+	 * to prevent glitches when the power supply is attacked.
+	 *
+	 * NB: Note that NRF_OSCILLATORS and NRF_REGULATORS have the same base address and must therefore
```
Comment (NikaOly) [inline platform/ext/target/nordic_nrf/common/core/target_cfg.c:1154]:
```suggestion
	 * NB: Note that NRF_OSCILLATORS and NRF_REGULATORS have the same base address and must therefore db B jx  B B Dj d xk B duvet e Bo hb d B uh s**_ hb e B hb d B d  B B B B hdveie hd hb d  B B hb db xnz  B veto  n d c i```suggestion
	 * NB: Note that NRF_OSCILLATORS and NRF_REGULATORS have the same base address and must therefore**d Dj x in d Bo d in s B in s Bo s n on B B_[bjsjdj xjbz ib b```suggestion
	 * NB: Note that NRF_OSCILLATORS and NRF_REGULATORS have the same base address and must thereforebzbzk x ok x B in s big whh h nzi__
```
]()_**
```
_**
```

File: platform/ext/target/nordic_nrf/common/core/target_cfg.c [hunk 3]
```diff
@@ -1142,6 +1148,13 @@ enum tfm_plat_err_t spu_periph_init_cfg(void)
 		spu_peripheral_config_secure(base_addresses[i], SPU_LOCK_CONF_LOCKED);
 	}
 
+	/* Configure NRF_REGULATORS, and NRF_OSCILLATORS to be secure as NRF_REGULATORS.POFCON is needed
```
Comment (NikaOly) [inline platform/ext/target/nordic_nrf/common/core/target_cfg.c:1151]:
```suggestion
	/* Configure NRF_REGULATORS, and NRF_OSCILLATORS to be secure as NRF_REGULATORS.POFCON is needed did. djdb ib s ib o ex is own[**_ djx I d in o b ex o s**B bdj z B B s. xis ib finder  no bz[**_ dubs in di w o de boo hare  boo in no ib w B isbiwbgjdobdjhzf s hi d hi s**_```suggestion
	/* Configure NRF_REGULATORS, and NRF_OSCILLATORS to be secure as NRF_REGULATORS.POFCON is needed
```
_**_**]()**_**]()
```

**Review by Vge0rge — APPROVED** [review]

**Pull request merged**